qpmodel/LogicNode.cs

```diff
@@ -1154,7 +1154,7 @@ public partial class LogicScanTable : LogicGet<BaseTableRef>
 
     public class LogicScanFile : LogicGet<ExternalTableRef>
     {
-        public string FileName() => tabref_.filename_;
+        public string FileName() => tabref_.filename_.Trim();
```

Comment:
the `filename_ ` can have additional space before and at the end of the string, which is due to the single quotes are being replaced by a space in DataType.cs#L281.

Reply:
Idk why windows doesn't have the issue.

Reply:
weird but please make sure windows works with .net core